src/Core/src/Fonts/FontManager.Android.cs

```diff
@@ -130,6 +135,21 @@ public FontSize GetFontSize(Font font, float defaultFontSize = 0)
 			return null;
 		}
 
+		Typeface? LoadDefaultTypeface(string fontfamily)
+		{
+			switch (fontfamily.ToLowerInvariant())
+			{
+				case "monospace":
+					return Typeface.Monospace;
+				case "sansserif":
+					return Typeface.SansSerif;
+				case "serif":
+					return Typeface.Serif;
+				default:
+					return null;
+			}
+		}
+
 		Typeface? CreateTypeface((string? fontFamilyName, FontWeight weight, bool italic) fontData)
```

Comment:
I wonder if this method is the correct place? `CreateTypeface`

The `LoadTypefaceFromAsset` is very specific so maybe moving this to check before - in the caller?

Reply:
Good point, I have reviewed the code and I think the changes are now in the right place.